distribution.properties

```diff
@@ -34,3 +34,4 @@ docs-ver=9.2.5
 #utils
 tango_admin=Release_1.14
 #mtangorest.server=rc4-2.9
+tango-doc=master
```

Comment:
Wouldn't it make sense to create a branch 9.3.3 in tango-doc git repo and point to that branch here instead of master?

Reply:
Sure it will. But I'm not sure if tango-doc is ready for branchout (see: tango-controls/tango-doc#276).

Reply:
I've changed this to 9.3.3 after branch was created in tango-doc.